src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -659,6 +674,16 @@ private SmugglerProgressBase.Counts ProcessCounters(SmugglerResult result)
                     if (result.Counters.ReadCount % 1000 == 0)
                         AddInfoToSmugglerResult(result, $"Read {result.Counters.ReadCount:#,#;;0} counters.");
 
+                    using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
```

Comment:
We need to move this inside the `actions.WriteCounter` for DatabaseDestination.

You can throw `DocumentDoesNotExist` exception there, catch it here and increase the errored count.